Foundation/NSURLSession/EasyHandle.swift

```diff
@@ -191,22 +188,10 @@ extension _EasyHandle {
     }
     /// Wait for pipelining/multiplexing
     /// - SeeAlso: https://curl.haxx.se/libcurl/c/CURLOPT_PIPEWAIT.html
-    //func set(waitForPipeliningAndMultiplexing flag: Bool) {
-    //    try! CFURLSession_easy_setopt_long(rawHandle, CFURLSessionOptionPIPEWAIT, flag ? 1 : 0).asError()
-    //}
```

Comment:
May make sense to document what the `CURLOPT_PIPEWAIT` option helps achieve here ..